ifupdown2/addons/batman_adv.py

```diff
@@ -84,224 +85,223 @@ class batman_adv(Addon, moduleBase):
-        if ifaceobj.get_attr_value_first ('batman-ifaces'):
+    def _is_batman_device(self, ifaceobj):
+        if ifaceobj.get_attr_value_first('batman-ifaces'):
             return True
         return False
 
-    def _get_batman_ifaces (self, ifaceobj ):
-        batman_ifaces = ifaceobj.get_attr_value_first ('batman-ifaces')
+    def _get_batman_ifaces(self, ifaceobj):
+        batman_ifaces = ifaceobj.get_attr_value_first('batman-ifaces')
         if batman_ifaces:
-            return sorted (batman_ifaces.split ())
+            return sorted(batman_ifaces.split())
         return None
 
-    def _get_batman_ifaces_ignore_regex (self, ifaceobj):
-        ifaces_ignore_regex = ifaceobj.get_attr_value_first ('batman-ifaces-ignore-regex')
+    def _get_batman_ifaces_ignore_regex(self, ifaceobj):
+        ifaces_ignore_regex = ifaceobj.get_attr_value_first('batman-ifaces-ignore-regex')
         if ifaces_ignore_regex:
-            return re.compile (r"%s" % ifaces_ignore_regex)
+            return re.compile(r"%s" % ifaces_ignore_regex)
         return None
 
-    def _get_batman_attr (self, ifaceobj, attr):
+    def _get_batman_attr(self, ifaceobj, attr):
         if attr not in self._batman_attrs:
-            raise ValueError ("_get_batman_attr: Invalid or unsupported B.A.T.M.A.N. adv. attribute: %s" % attr)
+            raise ValueError("_get_batman_attr: Invalid or unsupported B.A.T.M.A.N. adv. attribute: %s" % attr)
 
-        value = ifaceobj.get_attr_value_first ('batman-%s' % attr)
+        value = ifaceobj.get_attr_value_first('batman-%s' % attr)
         if value:
             return value
 
         return None
 
-    def _read_current_batman_attr (self, ifaceobj, attr, dont_map = False):
-	# 'routing_algo' needs special handling, D'oh.
+    def _read_current_batman_attr(self, ifaceobj, attr, dont_map=False):
+        # 'routing_algo' needs special handling, D'oh.
         if dont_map:
             attr_file_path = "/sys/class/net/%s/mesh/%s" % (ifaceobj.name, attr)
         else:
             if attr not in self._batman_attrs:
-                raise ValueError ("_read_current_batman_attr: Invalid or unsupported B.A.T.M.A.N. adv. attribute: %s" % attr)
+                raise ValueError("_read_current_batman_attr: Invalid or unsupported B.A.T.M.A.N. adv. attribute: %s" % attr)
 
             attr_file_name = self._batman_attrs[attr]['filename']
             attr_file_path = "/sys/class/net/%s/mesh/%s" % (ifaceobj.name, attr_file_name)
 
         try:
             return self.read_file_oneline(attr_file_path)
         except IOError as i:
-            raise Exception ("_read_current_batman_attr (%s) %s" % (attr, i))
+            raise Exception("_read_current_batman_attr (%s) %s" % (attr, i))
         except ValueError:
-            raise Exception ("_read_current_batman_attr: Integer value expected, got: %s" % value)
+            raise Exception("_read_current_batman_attr: Integer value expected, got: %s" % value)
```

Comment:
My IDE still marks value as unresolved reference - is this ment to be attr?

Reply:
@BarbarossaTM would you mind reviewing the PR as well, since you own most of the code here :)

Reply:
I think @moepman is right here, it seems this Exception never fired :)

I don't like the style changes very much, but if it's in compliance with the rest of the code I guess they are ok.

Reply:
@BarbarossaTM the goal is to be PEP8 compliant which will require those changes. 
I will still need to add some automatic code coverage and code-checking tools. 

Thanks for reviewing this!

Reply:
So I am still unsure as to where the ValueError should even come from: the `read_file_oneline` implementation will never try to cast to an integer in the first place?